libs/common-services/src/test/java/uk/gov/di/ipv/core/library/fixtures/VcFixtures.java

```diff
@@ -910,7 +910,6 @@ static VerifiableCredential vcF2fBrp() {
                                 List.of(
                                         TestVc.TestEvidence.builder()
                                                 .txn("some-uuid")
-                                                .type(CRI_STUB_CHECK_EVIDENCE_TYPE)
```

Comment:
This needs to be double-checked, not sure why this was overriding the default (`IDENTITY_CHECK_EVIDENCE_TYPE`). Using the default type specified on the TestVc.TestEvidence class passes the test